.github/workflows/deploy-integ-appstream-egress.yml

```diff
@@ -36,6 +36,10 @@ jobs:
           go-version: 1.13
```

Comment:
On line 33 and the other lines that specify node-version: 12, let's use node v16.
https://github.com/actions/setup-node

Reply:
Updated!